File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/OccupationalAssessmentFormViewModel.ts
//$65742886
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { OccupationalAssessmentForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class OccupationalAssessmentFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.OccupationalAssessmentFormViewModel, Core';
    @Type(() => OccupationalAssessmentForm)

    public _OccupationalAssessmentForm: OccupationalAssessmentForm = new OccupationalAssessmentForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/OccupationalAssessmentFormViewModel.cs
//$B67451A3
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class OccupationalAssessmentFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class OccupationalAssessmentFormViewModel: BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var occupationalAssessmentFormService = new OccupationalAssessmentFormServiceController();
            occupationalAssessmentFormService.OccupationalAssessmentFormInternal(this, objectContext);
            return (OccupationalAssessmentForm)objectContext.GetObject(this._OccupationalAssessmentForm.ObjectID, "OccupationalAssessmentForm");
        }
    }
}
